doc/administrator/administrate.rst

```diff
@@ -27,8 +27,9 @@ All the layers in the admin interface have the following attributes:
  *  ``Restrictions area``: the areas through which the user can see the layer.
  *  ``Related Postgres table``: the related postgres table,
     used by the :ref:`administrator_editing`.
- *  ``Attributes to exclude``: the list of attributes that shouldn't appear in
-    the :ref:`administrator_editing` so that they cannot be modified by end user.
+ *  ``Attributes to exclude``: the list of attributes (database column) that shouldn't appear in
+    the :ref:`administrator_editing` so that they cannot be modified by the end user,
+    for the enumeration attribute the column name should end with '_id'.
```

Comment:
I would move this last line to a distinct sentence:
``For enumeration (enumerable?) attributes, the column name should end with '_id'.``